core/test/element_array_view_test.cpp

```diff
@@ -387,6 +387,18 @@ TEST(ElementArrayViewTest, slicing_view_of_view_collapse_and_broadcast) {
                      {1, 1, 13, 13}));
 }
 
+TEST(ElementArrayViewTest,
+     slicing_view_of_view_collapse_and_broadcast_ignore_dim) {
```

Comment:
This test was for functionality that you removed later? Also need to be reverted here?